ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/deprovision/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- jianlinliu
+- yunjiang29
+- gpei
+reviewers:
+- jianlinliu
+- yunjiang29
+- gpei
```

Comment:
add you as FIPS owner, so that you can review/approve any  changes under this folder

Reply:
@xiaojiey ^